editor-setup/vscode.md

```diff
@@ -0,0 +1,112 @@
+# Recommended VSCode Configurations
+
+## BuoyRails
```

Comment:
These repo specific mentions feel a little out of place IMO. I wonder if they fit better in a repo's wiki.

Reply:
Oh good catch, I overlooked that. Wonder if the guide could be tweaked to stay with suggestions on helpful plugins removing the reference to the repos

Reply:
I added a commit below and made the titles more general. Please take a look.